src/loc/statement/SofTermsAndConditionsFR.tsx

```diff
@@ -0,0 +1,171 @@
+                </p>
+                <p>Should an additional license exist between the parties that shall apply to the subject of this Collection Item, the parties agreed that the LITC supersedes in case of conflict.</p>
+                <ul>
+                    {
+                        logionClassification.transferredRights.map((right, index) => (
+                            <li key={ index }>
+                                <span>{ DESCRIPTIONS[right.code].shortDescription } ({ right.code }): { DESCRIPTIONS[right.code].description }</span>
+                                {
+                                    right.code === "REG" &&
+                                    <>
+                                        <br/>
+                                        <span className="recorded-data">&gt; Recorded data: { regionalLimit.join(" - ") }</span>
+                                    </>
+                                }
+                                {
+                                    right.code === "TIME" &&
+                                    <>
+                                        <br/>
+                                        <span className="recorded-data">&gt; Recorded data: <InlineDateTime dateTime={ logionClassification.expiration } dateOnly={ true } /></span>
+                                    </>
+                                }
+                            </li>
+                        ))
+                    }
+                </ul>
+                </>
+            }
+
+            {
+                specificLicences.length === 0 &&
+                <>
+                <p>Additional licensing terms / contract</p>
+                <p>None</p>
+                </>
+            }
+            {
+                specificLicences.map(element => (
+                    <>
+                    <p>Additional licensing terms / contract</p>
+                    <p>The Requester provided an additional specific contract with regards to the Collection Item Underlying Asset. This contract has been recorded in a LOC with the following ID:</p>
+                    <p>{ element.tcLocId.toDecimalString() }</p>
+                    </> 
+                ))
+            }
+        </div>
+    );
+}
+
+function getLogionClassification(item: SofCollectionItem): LogionClassification | undefined {
+    const element = item.termsAndConditions.find(element => element.type === "logion_classification");
+    if(element) {
+        return LogionClassification.fromDetails(element.tcLocId, element.details);
+    } else {
+        return undefined;
+    }
+}
+
+function getSpecificLicenses(item: SofCollectionItem): SpecificLicense[] {
+    return item.termsAndConditions
+        .filter(element => element.type === "specific_license")
+        .map(element => SpecificLicense.fromDetails(element.tcLocId, element.details));
+}
+
+export const DESCRIPTIONS: Record<LogionTransferredRightCode, LogionTransferredRightDescription> = {
```

Comment:
I know it's temporary solution, but still, why not refers to `logionLicenseItems` in `LogionClassification`?

Reply:
Because before the end of the sprint, all values on the record will be translated to French.